internal/experiments/experiment.go

```diff
@@ -27,7 +27,7 @@ const (
 func init() {
 	// Each experiment constant defined above must be registered here as either
 	// a current or a concluded experiment.
-	registerCurrentExperiment(UnknownInstances)
+	registerConcludedExperiment(UnknownInstances, "Unknown instances are being rolled into a larger feature for deferring unready resources and modules, which will be available in Terraform Stacks.")
```

Comment:
I'd be interested to leave this running, so that we can still enable the behaviour in alpha builds outside of stacks.

Maybe we could add a new experiment called DeferredActions or something to cover everything, but I don't mind if we just kept UnknownInstances as everything.

We could then have the CLI set `DeferralAllowed = true` for plan opts where the root module has the experiment enabled. I think we need to support enabling this in some way for the provider integration tests.